cmd/insprd/operators/kafka/nodes/converter.go

```diff
@@ -98,6 +98,7 @@ func (no *NodeOperator) withBoundary(app *meta.App) k8s.ContainerOption {
 			parent, chName, _ := metautils.RemoveLastPartInScope(resolved)
 			ch, _ := no.memory.Channels().Get(parent, chName)
 			ct, _ := no.memory.Types().Get(parent, ch.Spec.Type)
+			env[boundary+"_BROKER"] = no.returnChannelBroker(resolved)
```

Comment:
Is this what informs de deployment of each channel's broker?

Reply:
Yessir

Reply:
Yes